### PR TITLE
Modified castle architecture to fit the new problem specifications.

### PR DESCRIPTION
**Pull request by kelzuriel**
Modified castle architecture to fit the new problem specifications. Updated starting location.

### CODE REVIEW

File: theMind.js [hunk 1]
```diff
@@ -3,35 +3,38 @@
 
 
 
-var castle = new Array(10);
+var castle = new Array(3);
```
Comment (kelzuriel) [inline theMind.js]: Changed number of lines from 10 to 3.

File: theMind.js [hunk 2]
```diff
@@ -3,35 +3,38 @@
...
 for (var i = 0; i < castle.length; i++) {
   castle[i] = new Array(10).fill('Empty');
 }
 
-castle[0][0] = "Samurai";
-castle[8][8] = "Katana";
+castle[1][1] = "Samurai";
```
Comment (kelzuriel) [inline theMind.js]: Changed Samurai and Katana position.

File: theMind.js [hunk 3]
```diff
@@ -3,35 +3,38 @@
...
+castle[1][6] = "Katana";
 
+castle[0][0] = "Wall";
```
Comment (kelzuriel) [inline theMind.js]: Changed walls pattern.

File: theMind.js [hunk 4]
```diff
@@ -64,11 +67,10 @@ var calculateShortestPath = function([startY, startX], castle) {
 };
 
 var locationStatus = function(position, castle) {
-  var castleSize = castle.length;
   var dft = position.distanceFromTop;
   var dfl = position.distanceFromLeft;
 
-  if(dfl < 0 || dfl >= castleSize || dft < 0 || dft >= castleSize) {
+  if(dfl < 0 || dfl >= 10 || dft < 0 || dft >= 3) {
```
Comment (kelzuriel) [inline theMind.js]: Changed search limits from 10 up to down, and 10 right toleft, to 3 up todown, and 10 right to left.

Comment (kelzuriel) [reply theMind.js]: Irina: What is a castle?

File: theMind.js [hunk 5]
```diff
@@ -111,4 +113,4 @@ var exploreInDirection = function(currentPosition, direction, castle) {
   return newPosition;
 };
 
-console.log(calculateShortestPath([0,0], castle));
+console.log(calculateShortestPath([1,1], castle));
```
Comment (kelzuriel) [inline theMind.js]: Changed starting location.

File: theMind.js (hunk 1 above)
Comment (kelzuriel) [reply theMind.js]: DRY line and column number should only be changed in one place.

File: theMind.js (hunk 2 above)
Comment (kelzuriel) [reply theMind.js]: DRY Samurai position should only be changed in one place.

File: theMind.js [hunk 6]
```diff
@@ -2,38 +2,46 @@
 // before his daimio found out and made him cut his guts with his little sword...
 
 
+var lineNumber = 3;
```
Comment (dira) [inline theMind.js]: numărul cărei linii este ăsta?

**Review by dira — CHANGES_REQUESTED** [review]

**Pull request merged**